ui/frontend/index.scss

```diff
@@ -427,7 +427,7 @@ $PHI: 2.61803398875;
         margin-top: 10em;
 
         width: 600px;
-        height: 600px / $PHI;
+        height: 950px / $PHI;
```

Comment:
Why is this needed?

Reply:
I found that the background to the config popup would not fit the new rows, causing the "Done" button to be overlaid on top of the drop-downs as in issue #222.  I should probably have removed the divide by $PHI as the golden ratio is pretty well shot by this change.

Reply:
No issues with Intel vs AT&T, directives have the same format in both styles.  The [remove_assembler_directives](https://github.com/chandlore/rust-playground/blob/ec3518ca5887b75fc5ba2cb8d6982ac028eb3a20/ui/src/asm_cleanup.rs#L5-L19) function just searches for any lines that start with a period and excludes them from the updated string.